engine/crates/engine-v2/src/sources/graphql/federation.rs

```diff
@@ -151,6 +145,59 @@ impl FederationEntityResolver {
     }
 }
 
+async fn cache_fetches<'ctx, R: Runtime>(
+    ctx: ExecutionContext<'ctx, R>,
+    endpoint: schema::SchemaWalker<'_, GraphqlEndpointId>,
```

Comment:
just FYI, there is a `GraphqlEndpointWalker` alias

Reply:
rust analyzer doesn't do those though